src/test/java/spoon/test/position/PositionTest.java

```diff
@@ -1458,4 +1460,16 @@ public void testLambdaParameterPosition1() {
 				)
 		);
 	}
+	
+	@Test
+	@GitHubIssue(issueNumber = 4779, fixed = true)
```

Comment:
I don't fully understand this test case. It looks like it does not hit the modified code in `PositionBuilder`. Am I missing something?
(also, you're currently building the model twice, with the variables `build` and `classContent` being unused)

Reply:
I think I got a bit confused here and I agree that this test is not good
If that's OK I'll remove it